docs/sql-migration-guide.md

```diff
@@ -201,6 +222,58 @@ license: |
         </tr>
     </table>
 
+  - Since Spark 3.0, when casting interval values to string type, there is no "interval" prefix, e.g. `1 days 2 hours`. In Spark version 2.4 and earlier, the string contains the "interval" prefix like `interval 1 days 2 hours`.
+
+  - Since Spark 3.0, when casting string value to integral types(tinyint, smallint, int and bigint), datetime types(date, timestamp and interval) and boolean type, the leading and trailing whitespaces (<= ASCII 32) will be trimmed before converted to these type values, e.g. `cast(' 1\t' as int)` results `1`, `cast(' 1\t' as boolean)` results `true`, `cast('2019-10-10\t as date)` results the date value `2019-10-10`. In Spark version 2.4 and earlier, while casting string to integrals and booleans, it will not trim the whitespaces from both ends, the foregoing results will be `null`, while to datetimes, only the trailing spaces (= ASCII 32) will be removed.
+
+### SQL Query Statements
+
```

Comment:
Since it seems this section describes parser behaviours, how about `SQL Query Statements` -> `SQL Parsing`?

Reply:
To end users, it is hard to tell the difference between parsing and engine. Let me combine it to Query Engine.